python/pythonlab/neighborhood/neighborhood/__init__.py

```diff
@@ -0,0 +1 @@
+from .painter import Painter as Painter
```

Comment:
Nit: What do you think about naming the neighborhood subfolder `src`? At first I did a double take on seeing two `neighborhood` folder names in a row.

Reply:
I _think_ this is a python convention. The inner folder needs to be named `neighborhood` so that package is named `neighborhood`. Our other package [pycdo](https://github.com/code-dot-org/code-dot-org/tree/staging-next/python/pycdo) follows the same pattern.

Reply:
FWIW, I see an example of a python package [here](https://github.com/pypa/packaging/tree/main) which uses `<package-name>/src/<package-name>`. But I think consistency with an existing convention (`pycdo`) makes sense - thanks!